app/controllers/ErrorController.scala

```diff
@@ -56,7 +55,12 @@ class ErrorController @Inject()(
           logger.error(errorResponse.message)
           InternalServerError(serviceUnavailableView())
         },
-        spendData => Ok(howTaxIsSpentView(spendData, taxYear))
+        spendData =>
+          if (taxYear > appConfig.taxYear) {
+            InternalServerError(serviceUnavailableView())
```

Comment:
That will trigger a pagerduty. Can you use a forbidden status? Or a bad request?

Reply:
Done